l10n_it_vat_registries/vat_registry.py

```diff
@@ -152,19 +180,14 @@ def __init__(self, cr, uid, name, context):
 
     def set_context(self, objects, data, ids, report_type=None):
         self.localcontext.update({
-            'fiscal_page_base': data.get('fiscal_page_base'),
+            'registry_type': data['form'].get('registry_type'),
         })
-        return super(Parser, self).set_context(objects, data, ids, report_type=report_type)
-
-report_sxw.report_sxw('report.registro_iva_vendite',
-                       'registro_iva_vendite',
-                       'addons/l10n_it_vat_registries/templates/registro_iva_vendite.mako',
-                       parser=Parser)
-report_sxw.report_sxw('report.registro_iva_acquisti',
-                       'registro_iva_acquisti',
-                       'addons/l10n_it_vat_registries/templates/registro_iva_acquisti.mako',
-                       parser=Parser)
-report_sxw.report_sxw('report.registro_iva_corrispettivi',
-                       'registro_iva_corrispettivi',
-                       'addons/l10n_it_vat_registries/templates/registro_iva_corrispettivi.mako',
-                       parser=Parser)
+        return super(Parser, self).set_context(
+            objects, data, ids, report_type=report_type)
+
+
+class report_registro_iva_venidte(osv.AbstractModel):
```

Comment:
Ok it's just abstract class to embed old style report but ```report_registro_iva_vendite``` is preferable. Please fix the typo, thanks.